CHANGELOG.md

```diff
@@ -3,11 +3,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 # [v1.3.0](unreleased)
 ### Notes
-- This release supports for API 800,1000,1200  for resource listed below.
+- This release supports for API 800,1000,1200,1600  for resource listed below.
 
 ### Oneview Features supported
 - Hypervisor Manager
```

Comment:
Hypervisor manager and server certificate resources are not yet supported for API 1600 version, remove these entries as well.

Reply:
Done